Libraries/D3D12RaytracingFallback/src/GpuBVH2Builder.cpp

```diff
@@ -416,6 +422,13 @@ namespace FallbackLayer
         const UINT64 hierarchySize = ALIGN_GPU_VA_OFFSET(sizeof(HierarchyNode) * totalNumNodes);
         scratchMemoryPartitions.OffsetToHierarchy = totalSize;
         totalSize += hierarchySize;
+        
+        if (level == Level::Bottom)
```

Comment:
Can you make a helper function called SupportsTreeletReordering(Level level) that would make this both self-documenting and make it easy to catch all the points that need to be fixed when updating the top level AS's to support treelet reordering